professional-services/roles/head-of-engineering.md

```diff
@@ -0,0 +1,61 @@
+# Head of Engineering
+
+The Head of Engineering (HoE) oversees the engineering team that creates solutions within the Professional Services team. They are ultimately responsible for the output from the engineering team, and support them in their business as usual functions. They are the arbiters of engineering standards and interfacing with the rest of the business.
+
+## Responsibilities
+
+The HoE takes full responsibility for the engineering processes associated with creating solutions, from proposal to go-live and ongoing support.
+
+### Uphold modern standards
+
+Many engineers have their own styles of coding and preferred development approaches. While for minutia this is acceptable, the HoE works with the engineers to create a set of standards to be used across multiple solutions where appropriate. This includes things such as:
+
+- Defacto libraries
+- Linting
+- Tools
+
+These standards and approaches should be reviewed on a regular basis, to keep up with the fast-paced world of software engineering.
+
+### Keep ahead of the curve
+
+As the ways of doing things change, the engineering team will have to learn new skills, technologies, and patterns. The HoE is responsible for the processes allowing engineers to upskill themselves, and identifying situations when this must be done and ensuring the engineering team stay ahead of the curve.
+
+### Manage resourcing
+
+An engineering team can only be as effective as its headcount allows. The HoE works with the Head of Delivery to ensure utilisation is maximised, and to manage the recruitment and hiring of new team members when a need to do so is identified.
+
+### Be the interface to the engineering team
+
+The HoE will be the single point of contact for queries to/about the engineering team. This can take the form of [processes](#create-and-document-processes), but also in simply being available to field questions and participate in meetings where engineering input is required.
+
+### Be security-centric
+
+Our engineers must not only be security conscious, but have security front-of-mind when designing and delivering solutions. The HoE will be responsible for ensuring engineers are following correct security procedures, and that any solutions are security-centric.
+
+### Provide the right tools
+
+Good tool sets are an incredibly important part of allowing engineers to work at pace with minimal friction. The HoE will be responsible for ensuring the tools provided are relevant, and where one is not or does not exist, to procure one or to lead a team in building them.
+
+### Create and document processes
+
+The HoE will be responsible for enabling the engineering team to interface with the rest of the business in as frictionless a manner as possible. To this end they must create and document processes in partnership with other business functions that will allow this to happen; the end goal is that the HoE is not required to be "in the weeds" to solve business as usual tasks.
```

Comment:
The HoE will be responsible for enabling the engineering team to interface with the rest of the business with the least friction possible.